twill-yarn/src/test/java/org/apache/twill/yarn/ProvisionTimeoutTestRun.java

```diff
@@ -85,13 +105,29 @@ public TimeoutAction launchTimeout(Iterable<TimeoutEvent> timeoutEvents) {
         return TimeoutAction.recheck(10, TimeUnit.SECONDS);
       }
     }
+
+    @Override
+    public void aborted() {
+      try {
+        new File(context.getSpecification().getConfigs().get("parentFolderPath") + File.separator
```

Comment:
When constructing file, usually not to use `File.separator`, but instead use the constructor `File(File parent, String path)`.